typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.stories.tsx

```diff
@@ -434,6 +437,64 @@ Wells3d.parameters = {
     },
 };
 
+// Gullfaks wells.
+export const Wells3DGullfaks: ComponentStory<typeof SubsurfaceViewer> = (
```

Comment:
I think this story should have a more descriptive name, like "SimplifiedRendering"